src/schemas/Config.json

```diff
@@ -35,8 +35,8 @@
             "enum": ["parent", "peer", "child"]
           },
           "plugin": {
-            "description": "Name of the ILP plugin that should be used for this account.",
-            "type": "string"
+            "description": "Name or instance of the ILP plugin that should be used for this account.",
```

Comment:
Might be worth clarifying that you can only pass an instance when instantiating the connector from JavaScript. If I'm reading the docs from the perspective of environment variables I might get a bit confused trying to pass objects here.